app/build.gradle

```diff
@@ -187,6 +184,34 @@ dependencies {
     androidTestCompile 'com.android.support:support-annotations:25.1.0'
 }
 
+// SHA and timestamp caching courtesy of https://github.com/gdg-x/frisbee/blob/develop/app/build.gradle#L193-L218
+def commitSha() {
+    def file = new File(project.buildDir, "commit-sha.txt")
+    if (file.exists()) {
+        return file.text
+    }
+
+    def value = 'git rev-parse --short HEAD'.execute([], project.rootDir).text.trim()
```

Comment:
ah right, i misread a comment you had in a previous iteration ([here](https://github.com/kickstarter/android-oss/pull/92/commits/2f5281137a42a62c9914f2f9ab2cb7eddcb4cf6b)) and thought it was because of a gradle versioning thing that we didn't have access to the method

i think it's fine to keep this, as well as line 207, as is for now. if you have a windows machine to test and improve this on, that'd be great, but if not sadly we'll have to punt the support for windows for right now to move on with at least this build improvement